### ISSUE TITLE
"Collapse All But Explanation" should also uncollapse features if necessary

### ISSUE THREAD

**Issue by tthuem** [FeatureIDE/FeatureIDE#658]
The visualization of an explanation for a large feature model is almost impossible right now, as it requires to first uncollapse all features (which is not feasible for large models) and then to collapse all but explanation. 

This should be fixed on release branch and made available also to develop.

Besides that, the action should be renamed to "Collapse Features Irrelevant for Explanation" or something similar.

**Comment by tthuem** [comment]
@TimoGuenther Do want to fix that?

**Comment by TimoGuenther** [comment]
The expanding functionality of that action was already added to `explanations` and `develop` with https://github.com/FeatureIDE/FeatureIDE/commit/77c69a51247daaaf44d3899af0601368b8f55476, so I merged `explanations` into `release` with https://github.com/FeatureIDE/FeatureIDE/commit/b5db81a28439279e48a7c4f1e8a2723ecefb5d91.

As for the name of the action, I decided to use _Collapse All But Explanation_ out of concern for brevity. After all, this is displayed in the context menu, which would get substantially bigger. In any case, the action's name should be trivial to change if wanted.
